finagle/h2/src/main/scala/io/netty/handler/codec/http2/H2FrameCodec.scala

```diff
@@ -298,6 +298,22 @@ object H2FrameCodec {
         ctx.fireChannelRead(data)
         0 // bytes are marked as consumed via WindowUpdateFrame writes
       }
+
+      override def onPingRead(
+        ctx: ChannelHandlerContext,
+        data: Long
+      ): Unit = {
+        val ping = new DefaultHttp2PingFrame(data)
+        ctx.fireChannelRead(ping); ()
+      }
+
+      override def onPingAckRead(
```

Comment:
perhaps add a comment that we need to propagate ping acks so that we can inform the failure detector that an ack has been received.